ReplayRequests/ReplayRequests/Program.cs

```diff
@@ -0,0 +1,76 @@
+﻿using Microsoft.ServiceBus.Messaging;
+using Microsoft.WindowsAzure;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace ReplayRequests
+{
+    class Program
+    {  
+        public static void Main(string[] args)
+        {
+            try
+            {
+                using (var requestsFile = File.CreateText("Rquests.txt"))
+                {
+                    var connectionString = CloudConfigurationManager.GetSetting("Microsoft.ServiceBus.ConnectionString");
+                    var queueClient = QueueClient.CreateFromConnectionString(connectionString, CloudConfigurationManager.GetSetting("QueuePath"));
+                    var cancellationTokenSource = new CancellationTokenSource();
+                    var cancellationToken = cancellationTokenSource.Token;
+                    var task = ContinouslyReadFromQueue(cancellationToken, queueClient, requestsFile);
+                    Console.WriteLine("Press any key to exit...");
+                    Console.ReadKey();
+                    cancellationTokenSource.Cancel();
+                    task.Wait();
+                }
+            }catch(Exception e)
```

Comment:
What happens if there is an error in one of the requests, does the whole process die?

What's the point of catching the exception and printing it out? The console app already does it by default, and that can be now found through the machine logs. If you catch it, its gone